plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/app/DBPApplication.java

```diff
@@ -136,4 +136,24 @@ default boolean isCommunity() {
         return false;
     }
 
+    /**
+     * Whether this product can participate in the Early Access Program but no license is present
+     */
+    default boolean isEarlyAccessProgramCandidate() {
+        return false;
+    }
+
+    /**
+     * Whether this product is part of the Early Access Program
+     */
+    default boolean isEarlyAccessProgramActive() {
```

Comment:
Maybe in CE it should be isStatisticsCollectionRequired? Or move all stuff related to it into PRO